README.md

```diff
@@ -35,17 +35,20 @@ helm upgrade --install newrelic newrelic-prometheus/newrelic-prometheus-agent -f
 For further information of the configuration needed for the chart just read the [chart's README](/charts/newrelic-prometheus-agent/README.md).
 
 ## Getting Started
-The simplest way to start working with this project is to install the [Chart](/charts/newrelic-prometheus-agent/README.md) in your Kubernetes cluster which by default will scrape the Prometheus targets having the `'prometheus.io/scrape: true'` annotation.
+The simplest way to start working with this project is to install the [Chart](/charts/newrelic-prometheus-agent/README.md) in your Kubernetes cluster.
+
+All pod and endpoints with the `newrelic.io/scrape: true` annotation will be scraped by default.
+
+Moreover, the solution will scrape as well all pods and endpoints with the `prometheus.io/scrape: true` annotations and
+having one of the labels matching the integrations_filter configuration.
```

Comment:
```suggestion
have one of the labels matching the integrations_filter configuration.
```

Reply:
🤔 I believe in this case it is correct. 

it would be have if it was " and `that` have one of the..."